collector/src/toolchain.rs

```diff
@@ -123,7 +123,10 @@ impl SysrootDownload {
         let components = ToolchainComponents::from_binaries_and_libdir(
             sysroot_bin("rustc")?,
             Some(sysroot_bin("rustdoc")?),
-            Some(sysroot_bin("clippy")?),
+            match sysroot_bin("cargo-clippy") {
```

Comment:
There is a helper `or` method for this (https://doc.rust-lang.org/std/result/enum.Result.html#method.ok) in the stdlib, btw :) But you can keep it as it is here.

Reply:
Do I... *not know `std`* :fearful:. Haha I'll change it right now.